zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/elasticsearch/ElasticsearchEngineClient.java

```diff
@@ -84,6 +84,18 @@ public void createIndexTemplate(
       client.indices().putIndexTemplate(request);
       LOG.debug("Template [{}] was successfully created", templateDescriptor.getTemplateName());
     } catch (final IOException | ElasticsearchException e) {
```

Comment:
🔧 please split the `catch` clause to two if we have unique handling for one of exceptions.